CognizantMickey/generator-studio-api
# F/1 docker support

### What does this PR do?
Adds a Dockerfile and docker-compose.yml; allows the dev to run the application in Docker with a containerized DB! 

### Where should someone start to review?
1. Clone this repo
1. `npm uninstall -g generator-studio-api` to remove any preexisting installation
1. Run npm link
1. From ./README.md, follow the instructions to scaffold a new application 
1. Review app/templates/Dockerfile
1. Review app/templates/docker-compose.yml

### How could you test this PR's functionality?
1. Confirm that Docker and Docker-Compose are installed and up-to-date
1. Run the application with instructions in app/templates/README.md
1. Confirm that docs are surfaced at http://localhost/docs

### New dependencies?
None

### Gif of your feels
![](https://media.giphy.com/media/mXnO9IiWWarkI/giphy.gif)
